src/js/locale/pl.yaml

```diff
@@ -0,0 +1,532 @@
+    encryption:
+      label: Szyfrowanie
+      sublabel: Używaj SSL
+      description: Wymaga proxy SSL
+    switch: Przełącz do tego serwera
+    reconnect: Połącz ponownie
+  services:
+    title: Usługi
+    limited_access: Ograniczony dostęp
+    unknown: Nieznany
+    current_user: Aktualny użytkownik
+    authorization: Autoryzacja
+    refresh_token: Odśwież token
+    authorized: Zautoryzowano
+    read_only: Tylko do odczytu
+    unauthorized: Niezautoryzowano
+    enabled: Odblokowany
+    disabled: Zablokowany
+    connected: Połączony
+    disconnected: Rozłączony
+    spotify:
+      mopidy_spotify_not_running: Rozszerzenie Mopidy-Spotify nie jest uruchomione - nie będziesz mógł odtwarzać ścieżek ze Spotify
+      country:
+        label: Kraj
+        description: Kod kraju ISO 3166-1 alpha-2 (np. 'PL')
+      locale:
+        title: Lokalizacja
+        description: Kod językowy ISO 639 oraz ISO 3166-1 alpha-2 połączone podkreślnikiem (np. 'pl_PL')
+    icecast:
+      enable:
+        label: Odblokowane
+        description: Strumieniuj audio do tej przeglądarki
+      location:
+        label: Lokacja
+        description: Pełny URL twojego endpointu
+user:
+  title: Użytkownik
+  title_window: '%{name} (user)'
+  you: Ty
+discover:
+  title: Odkrywaj
+  categories:
+    title: 'Gatunek / Nastrój'
+  category:
+    title: 'Gatunek / Nastrój'
+    category: Kategoria
+  featured:
+    title: Proponowane playlisty
+  new_releases:
+    title: Nowości
+  recommendations:
+    title: Odkrywaj
+    tracks: Utwory
+    artists: Wykonawcy
+    albums: Albumy
+    body_title: Eksploruj nową muzykę
+    body_subtitle: Dodaj ziarna oraz właściwości muzyki by osiągnąć swoje brzmienie
+    too_many_seeds: Za dużo ziaren! Możesz użyć maksymalnie 5 utworów, wykonawców i gatunków.
+    find_recommendations: Znajdź rekomendacje
+library:
+  albums:
+    title: Moje albumy
+  artists:
+    title: Moi wykonawcy
+  browse:
+    title: Przeglądaj
+  browse_directory:
+    title: Katalog
+  playlists:
+    title: Moje playlisty
+  tracks:
+    title: Moje utwory
+modal:
+  add_to_playlist:
+    title: Dodaj do playlisty
+    subtitle: 'Wybierz playlistę do dodania %{count} utworów'
+    no_playlists: Brak dostępnych playlist
+    no_editable_playlists: Brak edytowalnych playlist
+  add_to_queue:
+    title: Dodaj do kolejki
+    subtitle: Dodaj listę URI (oddzielonych przecinkami) do kolejki odtwarzania
+    uris: URI
+    add_random: Znajdź losowe utwory
+    position:
+      label: Pozycja
+      end: Koniec
+      next: Po aktualnym utworze
+  edit_playlist:
+    title: Edytuj playlistę
+    title_create: Stwórz playlistę
+    name_required: Nazwa jest wymagana
+    name: Nazwa
+    description: Opis
+    provider: Dostawca
+    image:
+      label: Okładka
+      description: Tylko JPEG, 256kB maks. Zostaw puste by pozostawić okładę bez zmian.
+    options:
+      label: Opcje
+      public: Publiczne
+      collaborative: Kolaboracja

```

Comment:
I suggest `Współpraca` as `Kolaboracja` is considered negative by some sources. Also I was looking for an adjective form but couldn't find one.